sklearn/tests/test_random_projection.py

```diff
@@ -365,8 +378,10 @@ def test_johnson_lindenstrauss_min_dim():
     assert johnson_lindenstrauss_min_dim(100, eps=1e-5) == 368416070986
 
 
+@pytest.mark.parametrize("coo_container", COO_CONTAINERS)
```

Comment:
Indeed, I was thinking to not parametrize since the test is not really intended for that. But since not all the CIs support arrays, we cannot use them by default so this is best to parametrize as you did. Sorry for the confusion.

Reply:
No worries, I'm grateful for your support.